GettingStarted.md

```diff
@@ -23,31 +23,28 @@ This project has a few system dependencies.
   ```$ cd myFolder/playground```  
     
   Setup the virtual environment for this specific project  
-  ```$ virtualenv venv```  
+  ```$ virtualenv venv```
     
   Activate the virtual environment  
   ```$ source venv/bin/activate```  
 
   Now you want to install project dependancies, before doing so make sure the virtual environment is started  
   you should see a (venv) at the beginning of terminal prompt.
-  ```$ pip install -r requirements.txt```
-
-  Once everything is installed, move to games folder using this command :y
-  ```$ cd games```
+  ```$ pip install .```
```

Comment:
This should specify from which file to install. There are currently two - one that has the requirements without ipython, etc, and one that has them with ipython.